datadog_checks_dev/datadog_checks/dev/_env.py

```diff
@@ -92,14 +92,19 @@ def format_config(config):
     return config
 
 
-def replay_check_run(agent_collector, stub_aggregator):
+def replay_check_run(agent_collector, stub_aggregator, stub_agent):
     errors = []
     for collector in agent_collector:
         aggregator = collector['aggregator']
+        inventories = collector.get('inventories')
```

Comment:
Did we call it that way in the agent? :/

Reply:
Yes, that's not great to have a different name for `agent check zk` (metadata) and `agent check zk --json` (inventories).

Reply:
Yeah, not sure what to do about that. Internals of the datadog-agent is using 'inventories' everywhere